apps/portal/src/content/docs/components/split-button.mdx

```diff
@@ -254,6 +278,20 @@ return (
       value: "boolean",
       defaultValue: "false",
     },
+    {
+      name: "disableButton",
+      description: "Disables the button while dropdown is enabled.",
+      required: false,
+      value: "boolean",
+      defaultValue: "false",
+    },
+    {
+      name: "disableDropdown",
+      description: "Disables the dropdown while button is enabled.",
+      required: false,
+      value: "boolean",
+      defaultValue: "false",
+    },
```

Comment:
These are the new additions in `SplitButton` to disable `Button` and `Dropdown` separately.